docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -713,6 +713,15 @@ You can also inject named `Tokens`, see <<named-oidc-clients,Inject named OidcCl
 `OidcClientRequestReactiveFilter`, `OidcClientRequestFilter` and `Tokens` producers will refresh the current expired access token if the refresh token is available.
 Additionally, the `quarkus.oidc-client.refresh-token-time-skew` property can be used for a preemptive access token refreshment to avoid sending nearly expired access tokens that might cause HTTP 401 errors. For example, if this property is set to `3S` and the access token will expire in less than 3 seconds, then this token will be auto-refreshed.
 
+By default, Quarkus will refresh the current expired token when you request tokens.
+Performance critical applications may want to avoid this waiting during incoming requests and configure asynchronous token refresh like in the example below:
```

Comment:
```suggestion
Performance critical applications may want to avoid having to wait for a possible token refresh during the incoming requests and configure an asynchronous token refresh instead, for example:
```